js/index.js

```diff
@@ -1 +1,65 @@
+//Corey Gumbs
+//Sept 18, 2018
 // Your code goes here
+
+
+
+///Navigation
+const aLinks = document.querySelectorAll("a");
+aLinks.forEach((link) => {
+
+    link.addEventListener("mouseover", (event) => {
+        event.target.style.color = "#17A2B8";
+    });
+
+    link.addEventListener("mouseleave", (event) => {
+        event.target.style.color = "#212529";
+    });
+});
+
+//navigation line preventDefault
+const navigation = document.querySelector(".nav a");
+
+navigation.setAttribute("href", "http://LambdaSchool.com");
+
+navigation.addEventListener("click", (event) => {
+    event.preventDefault();
```

Comment:
Hmm. Seems like this will never take me to LambdaSchool since you are preventDefault()